webpack.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  entry: './src/assets/scripts/script.js',
+  output: {
+    filename: 'main.js',
+  },
+};
```

Comment:
- [ ] Webpack is not correctly configured, the output should be an HTML file, a CSS file, a JS file, and any assets you used in the project. Currently, this only outputs the JS file.